### ISSUE TITLE
ImportError: cannot import name 'Flag'

### ISSUE THREAD

**Issue by Arlodotexe** [OpenXbox/xbox-smartglass-core-python#3]
OS: Raspbian Wheezy
Python Version: Python 3.5
```

When running `xbox-discover` , `xbox-poweroff` or `xbox-poweron` (not `xbox-authenticate`,  I get the following error message:

```(env) pi@raspberrypi:~ $ xbox-poweroff
Traceback (most recent call last):
  File "/home/pi/.local/bin/xbox-poweroff", line 7, in <module>
    from xbox.sg.scripts.poweroff import main
  File "/home/pi/.local/lib/python3.5/site-packages/xbox/sg/scripts/poweroff.py", line 13, in <module>
    from xbox.sg.console import Console
  File "/home/pi/.local/lib/python3.5/site-packages/xbox/sg/console.py", line 40, in <module>
    from xbox.sg.crypto import Crypto
  File "/home/pi/.local/lib/python3.5/site-packages/xbox/sg/crypto.py", line 34, in <module>
    from xbox.sg.enum import PublicKeyType
  File "/home/pi/.local/lib/python3.5/site-packages/xbox/sg/enum.py", line 5, in <module>
    from enum import Enum, Flag
ImportError: cannot import name 'Flag'
```

**Comment by tuxuser** [comment]
README lists requirement: `Python >= 3.6` - you are using `3.5` though.
`Flag` is not implemented in `Py 3.5`, sorry.

**Comment by unlimitedcoder2** [comment]
I figured that out and forgot to close the issue but thanks anyway
